deno.jsonc

```diff
@@ -28,6 +28,6 @@
   },
   "lock": false,
   "tasks": {
-    "test": "deno fmt --check && deno lint && deno test --allow-read"
+    "test": "deno fmt --check && deno lint && deno test --allow-read --allow-none"
```

Comment:
Noticed [this was removed to be compatible with Deno 2](https://github.com/slack-samples/deno-give-kudos/pull/46#discussion_r1801159705) - super casual mention @filmaj

I think this is a good change to share across other samples (removing the flag from `deno.jsonc`) but will want to make sure we have tests in place or can otherwise avoid problems in CI before fanning out this removal 👻

Reply:
More context on the `allow-none` flag: https://github.com/denoland/deno/issues/24752

Reply:
Super interesting, thanks for sharing! It seems like `--permit-no-files` is the alternative, but I'm still hoping that we can include a test with all of our samples 😉